lib/Notes/NotesAccordion/components/NotesList/NotesList.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  FormattedDate,
+} from 'react-intl';
+import memoize from 'memoize-one';
+
+import { MultiColumnList } from '@folio/stripes-components';
+import {
+  noteShape,
+  columnShape,
+} from '../../NotesAccordion';
+
+export default class NotesList extends React.Component {
+  static propTypes = {
+    columnsConfig: PropTypes.arrayOf(columnShape),
+    notes: PropTypes.arrayOf(noteShape).isRequired,
+    onNoteClick: PropTypes.func.isRequired,
+  }
+
+  getItems() {
+    return this.props.notes
+      .map(note => {
```

Comment:
I belive it should be decomposed to noteItem functional component

Reply:
Unfortunately, it returns only an object with data. So definitely, it's better to keep it in a function, but not in a component